smartautoclicker/src/main/res/values/strings.xml

```diff
@@ -63,6 +63,8 @@
     <string name="dialog_click_config_condition_title">Only if</string>
     <string name="dialog_click_config_delay_after_title">Delay before next click</string>
     <string name="dialog_click_config_duration_hint">ms</string>
+    <string name="dialog_click_config_stop_after_executions">Stop scenario after number of executions</string>
+    <string name="dialog_click_config_executions_hint">unlimited executions</string>
```

Comment:
I think something like "Don't stop the scenario" will be easier to understand from a user point of view. What do you think ?